Source/WebCore/platform/graphics/gstreamer/DMABufVideoSinkGStreamer.cpp

```diff
@@ -180,7 +180,8 @@ bool webKitDMABufVideoSinkIsEnabled()
     static std::once_flag s_flag;
     std::call_once(s_flag, [&] {
         const char* value = g_getenv("WEBKIT_GST_DMABUF_SINK_DISABLED");
-        s_disabled = value && (equalLettersIgnoringASCIICase(value, "true"_s) || equalLettersIgnoringASCIICase(value, "1"_s));
+        auto valueSpan = span(value);
```

Comment:
Does this `span` function work on null pointers?

Reply:
Yes it does:
```
inline std::span<const LChar> span8(const char* string)
{
    return { reinterpret_cast<const LChar*>(string), string ? strlen(string) : 0 };
}
```